flexget/plugins/input/anidb_feed.py

```diff
@@ -0,0 +1,479 @@
+
+        Adds those extra Entry fields if found:
+        'content_size'
+        'rss_pubdate'
+        'anidb_fileversion'     # fileversion aka v1-5
+        'anidb_fid'             # anidb file_id
+        'anidb_feed_added'
+        'anidb_feed_source'
+        'anidb_feed_resolution'
+        'anidb_feed_crc_status'
+        'anidb_feed_language'
+        'anidb_feed_group'
+        'anidb_feed_grouptag'
+        'anidb_feed_subtitle_language'
+        'anidb_feed_priority'
+        'anidb_feed_quality'
+        'anidb_feed_size'
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'url': {'type': 'string', 'format': 'url'},
+            'priority': {'type': 'string', 'enum': ['all', 'medium-high', 'high'], 'default': 'all'},
+            'valid_only': {'type': 'boolean', 'default': False},
+            'include': one_or_more({'type': 'string', 'enum': ANIDB_EXTRA_TYPES}, unique_items=True)
+        },
+        'additionalProperties': False,
+        'required': ['url'],
+    }
+
+    def prepare_config(self, config):
+        if 'include' in config and not isinstance(config['include'], list):
+            config['include'] = [config['include']]
+        return config
+
+    # notification update interval is 15 minutes
+    @cached('anidb_feed', persist='15 minutes')
+    def on_task_input(self, task, config):
+        config = self.prepare_config(config)
+        task.requests.add_domain_limiter(LIMITER)
+        # Create entries by parsing AniDB feed
+        log.verbose('Retrieving AniDB feed')
+        url = re.sub(r'&pri=[1-9]', '', config['url'])  # strip existing priority
+        priority_fragment = ''
+        if 'animedbfeed' in url:
+            priority = 0
+            if config['priority'] == 'medium-high':
+                priority = 1
+            elif config['priority'] == 'high':
+                priority = 2
+            priority_fragment = '&pri=%s' % priority
+        return self.fill_entries_for_url(url + priority_fragment, task, config)
+
+    @staticmethod
+    def is_valid_entry(entry, ns_prefix_name=''):
+        for key in VALIDATION_LIST:
+            key_name = ns_prefix_name + key
+            if not entry.get(key_name):  # int(0) is invalid
+                return False
+            elif entry[key_name] in ['Raw/Unknown', 'N/A', 'Unchecked']:
+                return False
+        return True
+
+    def get_value_via_regex(self, key, entry, re_string, group_nr=1):
+        result = None
+        if entry.get(key):
+            value_string = entry[key]
+            if isinstance(value_string, basestring) and not value_string.isspace():
+                match = re.search(re_string, value_string)
+                if match and match.group(group_nr) and not match.group(group_nr).isspace():
+                    result = match.group(group_nr)
+            else:
+                log.error('Expecting string for re lookup got: %s, tyep: %s' % (value_string, type(value_string)))
```

Comment:
That is proper way to use logging, it is possible to uniquely identify this error line. If you use string substition they all are different (think of hash of the first param). We don't use these capabilities though ATM, but it's better to start using it correctly nonetheless :)